README.md

```diff
@@ -9,7 +9,6 @@
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
   - [:card_index: Entity Relationship Diagram](#er-diagram)
-  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
```

Comment:
- [ ] To make your documentation be more descriptive and preofessional, kindly make sure you improve your `Getting started` section by adding all information needed by a user such as how to `clone` your project, `access` the cloned directory, `run tests`(if exists),... To help you I suggest you refer **[this example](https://gist.github.com/tresorsawasawa/4cde69cff1b40161b9de380f9891f415)**.